docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/are-there-some-technical-services-complexities-that-might-increase-efforts.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Are there some technical complexities, that might increase efforts like: non standard services, high traffic amount, VPN?
+description: This document allows you to assess if a project has technical complexities, that might increase efforts like: non standard services, high traffic amount, VPN.
+template: howto-guide-template
+---
+
+# Are there some technical complexities, that might increase efforts like: non standard services, high traffic amount, VPN?
+
+{% info_block infoBox %}
+
+Resources: Backend, DevOps
+
+{% endinfo_block %}
+
+## Description
+
+This information should be received as part of the [prerequisites](/docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/paas-assessment-prerequisites.html).
+and can be additionally clarified with the customer.
+
+Services mentioned in prerequisites have to be compared with the services from Spryker documentation to get the diff:
```

Comment:
What's to be done with this diff?

Reply:
All the custom services used on project must be replaced with analogs that we offer on Spryker Cloud or as an alternative we can setup VPN connections to services hosted on the customer side in case there is no alternative offered on Spryker Cloud or the customer doesn't want to replace a service, in this case, we are not responsible for services hosted on the customer side.